ts/packages/ts/src/commands/commands.test.ts

```diff
@@ -221,5 +222,105 @@ describe(`commands`, () => {
         await Promise.all(deletePromises);
       }
     });
+    describe('PipeMode', () => {
+      test('run EvalCommand and pipe to another, all keys should exists in export', async () => {
```

Comment:
```suggestion
      test('run EvalCommand and pipe to another, all keys should exist in export', async () => {
```

Reply:
Please notice these are TESTS. not messages to a client or user in any way.

<img width="722" alt="image" src="https://github.com/configu/configu/assets/129258356/249bb258-71e1-4ec8-a757-bf5e3b7e5944">